documentation/adoc/cluster-controller.adoc

```diff
@@ -230,7 +230,7 @@ Because Strimzi uses the [JMX exporter](https://github.com/prometheus/jmx_export
 on each node, the JSON string used for metrics configuration in the cluster ConfigMap reflects the related JMX exporter 
 configuration file. For this reason, you can find more information on how to use it in the corresponding GitHub repo.
 
-For more information on how metrics work, the related documentation is available link:../../metrics/METRICS.md[here]
+For more information on how metrics work, see <<_metrics>>
```

Comment:
The link isn't really about "how metrics work". I guess it should be more something like "For more information about using the metrics with Prometheus and Grafana, see ..."